mappings/net/minecraft/RainSpalshParticle.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_260 net/minecraft/RainSpalshParticle
```

Comment:
Typo? Also it's in the `net.minecraft` package?
```suggestion
CLASS net/minecraft/class_260 net/minecraft/client/entity/particle/RainSplashParticle
```

Reply:
Yep, that's a typo